rand_walker_node:
  ros__parameters:
    SPEED_STOP_LINEAR: 0.0f
    SPEED_STOP_ANGULAR: 0.0f
    SPEED_FORWARD_LINEAR: 0.3f
    SPEED_FORWARD_ANGULAR: 0.0f
    SPEED_TURN_LINEAR: 0.0f
    SPEED_TURN_ANGULAR: 0.3f
    OBSTACLE_DISTANCE_THRESHOLD: 1.0f